src/index.js

```diff
@@ -1,17 +1,28 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
+import { applyMiddleware, compose, createStore } from 'redux';
+import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
 
 import * as serviceWorker from './serviceWorker';
+import reducer from './store/reducers/reducer';
 
 import App from './App';
 
-import './index.css';
+import './assets/main.css';
```

Comment:
i dont like this implementation, in development we should be using our styl files instead the ones that are being generated on the assets, that should only for build purposes, so would be better to look at this.